sdk/fabric/Azure.ResourceManager.Fabric/src/autorest.md

```diff
@@ -55,6 +55,11 @@ acronym-mapping:
 
 rename-mapping:
   CheckNameAvailabilityResponse: FabricCheckNameAvailabilityResult
+  CheckNameAvailabilityContent: FabricCheckNameAvailabilityContent
```

Comment:
well some of the renamings are not working.
Please add
```
mgmt-debug:
  show-serialized-names: true
```
and then run `dotnet build /t:GenerateCode` to see the real key to rename them.